lib/experimental-default-theme.json

```diff
@@ -161,6 +161,54 @@
 						"size": 42
 					}
 				],
+				"fontStyles": [
+					{
+						"name": "Regular",
+						"slug": "normal"
+					},
+					{
+						"name": "Italic",
+						"slug": "italic"
+					}
+				],
+				"fontWeights": [
+					{
+						"name": "Ultralight",
+						"slug": "100"
+					},
+					{
+						"name": "Thin",
+						"slug": "200"
+					},
+					{
+						"name": "Light",
+						"slug": "300"
+					},
+					{
+						"name": "Regular",
+						"slug": "400"
+					},
+					{
+						"name": "Medium",
+						"slug": "500"
+					},
+					{
+						"name": "Semibold",
+						"slug": "600"
+					},
+					{
+						"name": "Bold",
+						"slug": "700"
```

Comment:
In the other presets we use different keys for slug and value. This approach is using the same keys for both. The idea of presets is we can change the value and the places using a given slug will automatically all use the new value. If we use the slug for the value the main objective of the preset is not accomplished as one can not change the value without changing the slug.
The inline style we store ends up containing the value anyway `var:preset|font-weight|100`. I guess removing presents from the fontWeight and fontStyle and just using values is the way to go.

Reply:
> In the other presets we use different keys for slug and value.

I can definitely update the presets to use both slug and value. It just seemed odd to me having both the slug and the value the same for the most part. 

> The inline style we store ends up containing the value anyway var:preset|font-weight|100. I guess removing presents from the fontWeight and fontStyle and just using values is the way to go.

The benefit I saw in using the presets that could be defined via the theme.json was that it also provided a means to customize the available styles or weights.